documentation/modules/upgrading/con-upgrade-paths.adoc

```diff
@@ -6,31 +6,33 @@
 = Strimzi upgrade paths
 
 [role="_abstract"]
-Two Strimzi upgrade paths are possible.
+Two upgrade paths are available for Strimzi.
 
 Incremental upgrade::
-Upgrading Strimzi from the previous minor version to version {ProductVersion}.
+An incremental upgrade involves upgrading Strimzi from the previous minor version to version {ProductVersion}.
 
 Multi-version upgrade::
-Upgrading Strimzi from an old version to version {ProductVersion} within a single upgrade (skipping one or more intermediate versions).
-+
-For example, upgrading from Strimzi 0.24.0 directly to Strimzi {ProductVersion}.
-+
-NOTE: A multi-version Strimzi upgrade might still support the current version of a Kafka deployment.
+A multi-version upgrade involves upgrading an older version of Strimzi to version {ProductVersion} within a single upgrade, skipping one or more intermediate versions. 
+For example, upgrading directly from Strimzi 0.24.0 to Strimzi {ProductVersion} is possible. 
```

Comment:
Upgrade to current Strimzi versions from 0.24 is not supported anymore (see https://strimzi.io/docs/operators/in-development/deploying.html#con-upgrade-paths-earlier-versions-str) => SO maybe we can use some other version. For example 0.30 would match the multi-version upgrade requirements.